editors/nvi/files/patch-dist_configure.diff

```diff
@@ -0,0 +1,11 @@
+--- dist/configure.orig	2023-09-23 00:19:17.000000000 -0700
++++ dist/configure	2023-09-23 00:20:05.000000000 -0700
+@@ -8231,7 +8231,7 @@
+              10.[012])
+                allow_undefined_flag='${wl}-flat_namespace ${wl}-undefined ${wl}suppress'
+                ;;
+-             10.*)
++             *)
```

Comment:
Please use a standard version of this patch. You can find various versions of it in the repository under the name dynamic_lookup-11.patch (as was already mentioned in the ticket). You'll have to find the version that applies to this vintage of configure script, or hand-apply it and regenerate it.

Reply:
The closest version of this patch that I could find was in the zthread port.  I took that version of the patch and modified it to work for this port in my latest commit.